flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/CreateTableConverterUtils.java

```diff
@@ -85,8 +74,10 @@ Operation convertCreateTable(SqlCreateTable sqlCreateTable) {
     }
 
     /** Convert the {@link SqlCreateTableAs} node. */
-    Operation convertCreateTableAS(
-            FlinkPlannerImpl flinkPlanner, SqlCreateTableAs sqlCreateTableAs) {
+    public static Operation convertCreateTableAs(
+            SqlNodeConverter.ConvertContext context, SqlCreateTableAs sqlCreateTableAs) {
+        CatalogManager catalogManager = context.getCatalogManager();
+        FlinkPlannerImpl flinkPlanner = context.getFlinkPlannerImpl();
```

Comment:
I'm trying not to expose `FlinkPlannerImpl` to the converters because it pulls out many complex objects. I think we can simply call `SqlNodeConverters#convertSqlNode(sqlCreateTableAs.getAsQuery(), context)` once we migrate query conversion.